monai/apps/nnunet/nnunetv2_runner.py

```diff
@@ -833,19 +833,20 @@ def predict(
             self.default_num_processes if num_processes_segmentation_export < 0 else num_processes_segmentation_export
         )
 
-        predict_from_raw_data(
-            list_of_lists_or_source_folder=list_of_lists_or_source_folder,
-            output_folder=output_folder,
-            model_training_output_dir=model_training_output_dir,
-            use_folds=use_folds,
-            tile_step_size=tile_step_size,
+        predictor = nnUNetPredictor(
             use_gaussian=use_gaussian,
             use_mirroring=use_mirroring,
             perform_everything_on_gpu=perform_everything_on_gpu,
             verbose=verbose,
+        )
+        predictor.initialise_from_trained_model_folder(
```

Comment:
thanks for lookint into the issue, this currently generates an error `AttributeError: 'nnUNetPredictor' object has no attribute 'initialise_from_trained_model_folder'. Did you mean: 'initialize_from_trained_model_folder'?`

I think you can test within a docker container (`docker run --ipc=host --net=host --gpus all -ti --rm  projectmonai/monai`):
```
cd /tmp
git clone --depth 1 --branch 7013nnunetv2-predict --single-branch https://github.com/JupiLogy/MONAI.git
python -m pip install git+https://github.com/julien-blanchon/hiddenlayer.git
python -m pip install nnunetv2
cd MONAI/
python -m tests.test_integration_nnunetv2_runner
```